aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -155,7 +155,13 @@ In some cases, NameValuePairs may be used as read-only display for text inputs t
 
 ### Content Truncation
 
-When a value content exceeds 192 characters, truncate content and include a Button to expand to see more.
+When presenting a list of items, the list may be truncated and 
+accompanied by a Button to expand the list, displaying all items.
+
+When delimitted, content exceeding 192 characters should be truncated. 
+
+As a vertical list, a list of 7+ items should be truncated, displaying the first 3 items
+with a "Show all" button; 6 or fewer items, show all items and do not truncate.
 
```

Comment:
Trying to simplify the language a little (feel free to take it or leave it as you see fit):

For vertical lists:
- 7+ items should be truncated, displaying the first 3 items with a "Show all" button.
- For 6 or fewer items, show all items and do not truncate.

Reply:
Done.